ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -860,6 +887,39 @@ export const AdminInstitutionEdit = fp.flow(
                     inputStyle={{ width: '16rem', marginTop: '0.8rem' }}
                   />
                 )}
+                <TextInputWithLabel
+                  value={institution.requestAccessUrl}
+                  inputId='requestAccessUrl'
+                  inputName='requestAccessUrl'
+                  placeholder='Request Access URL'
+                  containerStyle={{ marginTop: '1.5rem' }}
+                  labelStyle={styles.label}
+                  inputStyle={{ width: '16rem', marginTop: '0.3rem' }}
+                  labelText='Custom URL for “Request Access” Links'
+                  onChange={(v) =>
+                    this.setState(
+                      fp.set(['institution', 'requestAccessUrl'], v)
+                    )
+                  }
+                  onBlur={(v) =>
+                    this.setState(
+                      fp.set(['institution', 'requestAccessUrl'], v.trim())
+                    )
+                  }
+                />
+
+                <p
+                  style={{
+                    color: colors.primary,
+                    fontSize: 12,
+                    width: '16rem',
+                  }}
+                >
+                  If provided, users who select this institution but are not
+                  allowed according to the data access tier requirements below
+                  will be guided to the custom URL rather than the standard ones
+                  (initial registration, request for CT access).
```

Comment:
I think we don't use an abbreviation. `CT` is `Controlled Tier`.

Reply:
I agree